builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -394,8 +407,11 @@ func resourceVSphereVirtualMachineCreate(d *schema.ResourceData, meta interface{
 			} else {
 				if v, ok := disk["size"].(int); ok && v != 0 {
 					disks[i].size = int64(v)
+				} else if v, ok := disk["bootable_vmdk_path"].(string); ok && v != "" {
+					disks[i].vmdkPath = v
+					vm.bootableVmdkPath = v
 				} else {
-					return fmt.Errorf("Size argument is required.")
+					return fmt.Errorf("Size or bootable_vmdk_path argument is required.")
```

Comment:
Perhaps create an anonymous function for this and get rid of the code duplication.